### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by mariagus**
Please could you review my code! Thanks

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+}
+
+nav a {
+	text-decoration: none;
+	font-family: 'Nixie One', serif;
+	color: #a6e0d2;
+	margin-right: 5px;
+	margin-left: 5px;
+	padding: 10px;
+	border-style: none;
```
Comment (ashleycoles) [inline style.css]: This line doesn't seem to do anything for me

File: index.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+		       	 	<li>Express</li>
+		        	<li>MongoDB</li>
+		        	<li>Mongoose</li>
+			</ul>
+		</div>
+		        </div>
+	</section>
+	<section id="contact" class="contact">
+		<a class="contact-box" href="mailto:masha.gu@gmail.com"><h3 class="contact-font"><span>don't</span> get in touch.</h3></a>
+		<a class="git" href="https://github.com/mariagus"><img src="github.png" alt="github icon"/></a>
```
Comment (ashleycoles) [inline index.html:71]: I would suggest making this link open in a new tab, just to stop people from leaving your site if they click it! You can do that by giving the a tag a `target="_blank"` attribute

File: index.html [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+				<a href="https://github.com/firefox-devtools/profiler"><img src="profiler.png" alt="firefox profiler"/><p>Firefox Profiler</p></a>
+			</div>
+			<div class="project">
+				<a href="https://mariagus.github.io/de-tough/"><img src="detough.png" alt="DE-Tough"/><p>DE-Tough LTD</p></a>
+			</div>
+			<div class="project">
+				<a href="https://mariagus.github.io/grumpygit/"><img src="grumpygit.png" alt="Grumpy Git"/><p>Grumpy Git</p></a>
+			</div>
+			<div class="project">
+				<a href="https://mariagus.github.io/pomodoro-clock/"><img src="pomodoro.png" alt="Pomodoro Clock"/><p>Pomodoro Clock</p></a>
```
Comment (ashleycoles) [inline index.html]: I'd also suggest making these links open in a new tab using `target="_blank"` on the a tags

File: style.css [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+h3 {
+	font-size: 2rem;
+}
+
+.project:hover {
+	opacity: 0.5;
+}
+
+.about-me {
+	height: fit-content;
```
Comment (ashleycoles) [inline style.css]: Does this height have any effect?

File: index.html [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+	</header>
+	<main id="home">
+		<div class="top" id="fade">
+		<h1>maria</h1>
+		<h1>gusova.</h1>
+	</div>
+	<div class="bottom" id="fade">
+		<h2>software</h2>
+		<h2>developer.</h2>
+	</div>
```
Comment (ashleycoles) [inline index.html]: Duplicate ID here, suggest using a class instead as you can reuse them infinitely

File: index.html [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+	<link href="https://fonts.googleapis.com/css2?family=Nixie+One&display=swap" rel="stylesheet"/>
+</head>
+<body class="background">
+	<header>
+		<nav>
+			<a href="#home">home.</a>
+			<a href="#projects">projects.</a>
+			<a href="#about">about.</a>
+			<a href="#contact">contact.</a>
+		</nav>
```
Comment (ashleycoles) [inline index.html:17]: On the iphone 5/se (the smallest device I ever bother with) the nav menu is slightly cut off on the right hand side

Screenshot 2021-03-11 at 09 54 54

**Review by ashleycoles — CHANGES_REQUESTED** [review]
Just a few minor comments for you! Nice work!

**Comment by mariagus** [comment]
Hey @ashleycoles 
I've made the changes, do you need to re-review?
Thanks

**Review by ashleycoles — APPROVED** [review]
Spot on! Happy for you to merge this into main, well done!

**Pull request merged**